### PR TITLE
tools/dm: add simple-synchronization-scenario.md

### PR DESCRIPTION
**Pull request by yikeke**
Via docs: https://github.com/pingcap/docs/blob/master/tools/dm/simple-synchronization-scenario.md
PTAL. @GregoryIan @csuzhangxc @lilin90

### CODE REVIEW

File: tools/dm/simple-synchronization-scenario.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
+---
+title: Data Migration 简单使用场景
+category: tools
+---
+
+# Data Migration 简单使用场景
+
+本文介绍了 DM 工具的一个简单使用场景（非分库分表场景）：将三个上游 MySQL 实例的数据同步到一个下游 TiDB 集群中。
```
Comment (GregoryIan) [inline tools/dm/simple-synchronization-scenario.md]: we use `分片合并` in some places https://github.com/pingcap/docs-cn/pull/1133/files, should we always use `分库分表合并`？ @dcalvin

File: tools/dm/simple-synchronization-scenario.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+
+1. 不合并 `user` 库。
+
+    1. 将实例 1 中的 `user` 库同步到下游 TiDB 的 `user_north` 库中。
+
+    2. 将实例 2 中的 `user` 库同步到下游 TiDB 的 `user_east` 库中。
+
+    3. 将实例 3 中的 `user` 库同步到下游 TiDB 的 `user_south` 库中。
+
+    4. 任何情况下都不删除 `log` 表。
```
Comment (GregoryIan) [inline tools/dm/simple-synchronization-scenario.md]:
```suggestion
    4. 任何情况下都不删除 `log` 表的任何数据。
```

File: tools/dm/simple-synchronization-scenario.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+2. 将上游 `store` 库同步到下游 `store` 库中，且同步过程中不合并表。
+
+    1. 实例 2 和实例 3 中都存在 `store_sz` 表，且这两个 `store_sz` 表分别被同步到下游的 `store_suzhou` 表和 `store_shenzhen` 表中。
+
+    2. 任何情况下都不删除 `store` 库。
```
Comment (GregoryIan) [inline tools/dm/simple-synchronization-scenario.md]:
```suggestion
    2. 任何情况下都不删除 `store` 库的任何数据。
```

**Comment by GregoryIan** [comment]
Rest LGTM

**Comment by GregoryIan** [comment]
LGTM

File: tools/dm/simple-synchronization-scenario.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+| Schema | Tables |
+|:------|:------|
+| user_north | information, log |
+| user_east  | information, log |
+| user_south | information, log |
+| store | store_bj, store_tj, store_sh, store_suzhou, store_gz, store_shenzhen |
+
+## 同步方案
+
+- 为了满足[同步要求](#同步要求)中第一点的前三条要求，需要配置以下[表路由规则](/tools/dm/data-synchronization-features.md#table-routing)：
```
[the remark this answers is not in the document]
Comment (yikeke) [reply tools/dm/simple-synchronization-scenario.md]: We use "table routing 规则". I'll change it.

File: tools/dm/simple-synchronization-scenario.md (hunk 1 above)
Comment (lilin90) [reply tools/dm/simple-synchronization-scenario.md]: @GregoryIan Yes, please always use `分库分表合并` @dcalvin

File: tools/dm/simple-synchronization-scenario.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (yikeke) [reply tools/dm/simple-synchronization-scenario.md]: Comment addressed. PTAL. @lilin90

File: tools/dm/simple-synchronization-scenario.md [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+        target-schema: "store"
+        target-table:  "store_suzhou"
+      instance-3-store-rule:
+        schema-pattern: "store"
+        table-pattern: "store_sz"
+        target-schema: "store"
+        target-table:  "store_shenzhen"
+    ```
+
+- 为了满足[同步要求](#同步要求)中第一点的第四条要求，需要配置以下 [binlog event filter](/tools/dm/data-synchronization-features.md#binlog-event-filter)：
```
Comment (csuzhangxc) [inline tools/dm/simple-synchronization-scenario.md]: how about
```suggestion
- 为了满足[同步要求](#同步要求)中第一点的第四条要求，需要配置以下 [binlog event filter 规则](/tools/dm/data-synchronization-features.md#binlog-event-filter)：
```

File: tools/dm/simple-synchronization-scenario.md [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+      ...
+      store-filter-rule:
+        schema-pattern: "store"
+        events: ["drop database", "truncate table", "drop table", "delete"]
+        action: Ignore
+    ```
+
+    > **注意**：`store-filter-rule` 不同于 `log-filter-rule` 和 `user-filter-rule`。`store-filter-rule` 是针对整个 `store` 库的规则，而 `log-filter-rule` 和 `user-filter-rule` 是针对 `user` 库中 `log` 表的规则。
+
+- 为了满足[同步要求](#同步要求)中的第三点要求，需要配置以下 [black-&-white-table-lists](/tools/dm/data-synchronization-features.md#black-&-white-table-lists)：
```
Comment (csuzhangxc) [inline tools/dm/simple-synchronization-scenario.md]: why use a `&` in `black-&-white-table-lists`? and whether needing to add a *规则* prefix?

File: tools/dm/simple-synchronization-scenario.md [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+        table-pattern: "log"
+        events: ["truncate table", "drop table", "delete"]
+        action: Ignore
+      user-filter-rule:
+        schema-pattern: "user"
+        events: ["drop database"]
+        action: Ignore
+    ```
+
+- 为了满足[同步要求](#同步要求)中第二点的第二条要求，需要配置以下[binlog event filter](/tools/dm/data-synchronization-features.md#binlog-event-filter)：
```
Comment (csuzhangxc) [inline tools/dm/simple-synchronization-scenario.md]: ditto

File: tools/dm/simple-synchronization-scenario.md (hunk 6 above)
Comment (yikeke) [reply tools/dm/simple-synchronization-scenario.md]: Ian suggests it. I'll add "规则" in it. Thanks!

File: tools/dm/simple-synchronization-scenario.md (hunk 7 above)
Comment (yikeke) [reply tools/dm/simple-synchronization-scenario.md]: Comment addressed. PTAL again. @csuzhangxc @lilin90

**Comment by csuzhangxc** [comment]
LGTM

**Pull request merged**